tests/roles/pull_openstack_configuration/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+dpa_dir: "/home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption"
+os_diff_data_dir: "/home/zuul-worker/src/github.com/openstack-k8s-operators/os-diff"
+standalone_ip: 192.168.122.100
```

Comment:
Sorry i didn't realize this in the original review, but this should probably be called something like `source_os_diff_config_ip` and it should go here in this file among the other similar variables:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/6be9a9f0888ab5dbc41c4ca90f0ae637663bd129/tests/vars.sample.yaml#L24

We don't want to specifically mention `standalone` in the variables because the test suite needs to work with 3-controller environments as well.